app/ui/lib/Badge.tsx

```diff
@@ -28,7 +28,7 @@ export const badgeColors: Record<BadgeVariant, Record<BadgeColor, string>> = {
     default: `ring-1 ring-inset bg-accent-secondary text-accent ring-[rgba(var(--base-green-800-rgb),0.15)]`,
     destructive: `ring-1 ring-inset bg-destructive-secondary text-destructive ring-[rgba(var(--base-red-800-rgb),0.15)]`,
     notice: `ring-1 ring-inset bg-notice-secondary text-notice ring-[rgba(var(--base-yellow-800-rgb),0.15)]`,
-    neutral: `ring-1 ring-inset bg-secondary text-secondary ring-[rgba(var(--base-neutral-700-rgb),0.15)]`,
+    neutral: `ring-1 ring-inset bg-secondary text-default ring-[rgba(var(--base-neutral-700-rgb),0.15)]`,
```

Comment:
neutral badge has lost its background color, not sure what that's about

<img width="1293" alt="image" src="https://github.com/user-attachments/assets/107aefca-72df-4bd0-b01d-00dbde9e9688">

Reply:
Fixed, it's because `bg-inverse-tertiary` no longer exists.